47 redaction partie interactivite

NicolasTrtrtt opened the pull request:
Rédaction chapitre interactivité OK pour moi, prêt à merge dans branche dev.

Commit:
rédaction partie Shiny x Rmarkdown

Commit:
Fin de la rédaction chap interactivité

jengelaere reviewed (commented):
Coucou Nicolas, chouette travail 👍, j'ai appris plein de trucs.
Pas eu le temps de compiler chez moi, mais deux petites questions :
- est-ce que ça te choque si j'enlève plotly Ligne 22, ça n'est plus détaillé dans le M5 je crois
- Est-ce que ça marche et est que ce serait ok pour toi si on met : 
`SharedData$new(dataframe) <- dataframe_partagee`
au lieu de 
`dataframe_partagee -> SharedData$new(dataframe)`
?
Comme on dit qu'il ne faut pas faire dans ce sens là dans le M1, de mémoire, je me dis qu'il vaut mieux être cohérent.
Enfin, le yml le l'exercice L106 m'a surprise, peut être qu'il y a deux coquilles, 
- il y a deux fois la date, celle à côté de author L108 est à supprimer ?
- celle sur deux lignes 114 et 115 fonctionne avec le retour à la ligne ? et lubricate ? coquille pour lubridate, j'imagine
A+

Commit:
corrections après relecture J.

NicolasTrtrtt commented:
- plotly enlevé
- correction de la flèche d'assignation, elle était complètement dans le mauvais sens, la bonne version est ```dataframe_partagee <- SharedData$new(dataframe)```
- le yaml L106 je l'ai copié-collé directement du chapitre 6 ligne 26 à 35 (j'ai juste ajouté le runtime shiny). Il y a eu disparition du saut de ligne après author, c'est corrigé. Pour la deuxième date, j'en comprends que c'est une variable interne passée en params, pas la date du document. Donc je pense qu'il ne s'agit pas d'un doublon. Le lubricate au lieu de lubridate est une faute de frappe qui était dans le chapitre 6 et que j'ai copié collé sans regarder...corrigé dans chap 8 et dans chap 6.

jengelaere committed:
Merge branch 'dev' into 47-redaction-partie-interactivite

jengelaere committed:
Qq modifs pour la compilation du bookdown :
- suppression des chunck avec md comme langage pour clarifier les logs de compilation
- supression de l'évaluation des chunks sur shiny (les 1ers étaient en erreur et bloquaient la compilation)
- renomage du fichier d'exercice de Caroline qui se placait au début
- ajout des nouvelles dépendances

jengelaere commented:
Coucou Nicolas @NicolasTrtrtt , merci pour les correctifs
j'ai procédé à qq ajustements pour que github parvienne à compiler le bookdown, est-ce que tu retrouves ce que tu attendais au niveau de ton chapitre dans le rendu html : https://github.com/MTES-MCT/parcours_r_module_publication_rmarkdown/actions/runs/13861554254/artifacts/2754658763 ? (à télécharger et dezipper)

NicolasTrtrtt commented:
Salut @jengelaere ,
Il y a un bug sur la fin de la page après le 11.5, il y a une dizaine de ligne de texte qui sont insérées et je ne comprend pas d'où elles viennent.
Il y a également plein de chapitres qui sortent de je ne sais où. On ne retrouve pas l'architecture des chapitres de 01 à 09.

jengelaere commented:
Coucou @NicolasTrtrtt c'est lié au fonctionnement de bookdown et au fait que ce soit le bazar dans le dépôt. On a plusieurs chapitres 7 ou 9 par exemple, donc bookdown renumérote sans doublon.
Les lignes en plus proviennent du fichier 09-aller_plus_moin_bis, ce qui est au dessus du premier titre de niveau 1 est rapatrié dans le chapitre précédent.

jengelaere committed:
Modif Rmd 09 aller plus loin bis : Déplacement du titre de niveau 1 pour que le chapitre html commence au niveau du début du fichier Rmd

Commit:
Correction exemple crosstalk (polygons impossibles)
